behave/docker-compose.yml

```diff
@@ -57,7 +63,7 @@ services:
       - "4444:4444"
     shm_size: 2g
     volumes:
-      - ./fox_admin_downloads:/fox_admin_downloads
+      - ./data:/data
```

Comment:
now this is a different location, I wonder if we can stop using user:root for this container?